### ISSUE TITLE
Releaser workflow missing

### ISSUE THREAD

**Issue by claes** [anacrolix/dms#154]
Hello, 
I have packaged DMS for NixOS/nixpkgs

I have  a question about DMS release process. The latest packaged release is 1.6.0 but there are later tags, up until 1.7.1.
What should be seen as the latest release, is there a stable/unstable concept or similar?

**Comment by anacrolix** [comment]
No, there are no intentionally unstable releases. Is there something that makes 1.6 appear to be stable? Is it a GitHub releases?

**Comment by anacrolix** [comment]
Ah, looks like I removed the releaser workflow as it was broken. I recommend to use the latest tag.

**Comment by claes** [comment]
Yes exactly. Will do, thank you!

**Comment by anacrolix** [comment]
Getting the releaser working is open to contribution!

**Comment by claes** [comment]
For nixpkgs, the tag is sufficient. But I understand that for other uses, a release tarball is of course useful.

**Comment by anacrolix** [comment]
Ah yeah I meant open to general contribution. I'll leave this issue open for other people that need other styles of release.
